SECURITY.md

```diff
@@ -1,55 +1,52 @@
-# A note on the security of the included implementations and schemes
+# PQClean Security Notice
 
-This project contains (reference) implementations of cryptographic libraries.
-We do not make any security claims about the code included in PQClean.
-In the current state, we distribute reference implementations with minor modifications.
-We did not perform any extensive security analyses.
-This code is suitable for experimental or scientific use.
-We recommend careful expert code review before using any of the included implementations in production environments.
+## Project Scope
 
-See also the [NIST PQC Forum][forum] for discussion about the cryptographic schemes included in PQClean.
+PQClean offers reference implementations of cryptographic algorithms, primarily for academic and experimental purposes. 
+It is important to understand that these implementations have not been subjected to rigorous security audits or formal security validations. 
```

Comment:
I prefer the original phrasing here, as we have more than just reference implementations (see the AVX2 and ARM specific code) and I prefer making it explicit that we barely change the upstream implementations.

Reply:
Okay, I'll work this into the change.